File: Basic/7.Find duplicates in an array/Solution.py
def find_duplicates(arr):
    print("Duplicate elements: ", end="")
    found = False

    for i in range(len(arr) - 1):
        for j in range(i + 1, len(arr)):
            if arr[i] == arr[j]:
                print(arr[i], end=" ")
                found = True
                break  # Avoid printing the same duplicate multiple times

    if not found:
        print("None")

# Input from the user
n = int(input("Enter the number of elements in the array: "))
arr = []

print("Enter the elements of the array:")
for _ in range(n):
    arr.append(int(input()))

find_duplicates(arr)

File: Basic/7.Find duplicates in an array/Solution.java
import java.util.Scanner;

public class FindDuplicates {

    static void findDuplicates(int[] arr) {
        System.out.print("Duplicate elements: ");
        boolean found = false;

        for (int i = 0; i < arr.length - 1; i++) {
            for (int j = i + 1; j < arr.length; j++) {
                if (arr[i] == arr[j]) {
                    System.out.print(arr[i] + " ");
                    found = true;
                    break; // Avoid printing the same duplicate multiple times
                }
            }
        }

        if (!found) {
            System.out.print("None");
        }
        System.out.println();
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        System.out.print("Enter the number of elements in the array: ");
        int n = scanner.nextInt();

        int[] arr = new int[n];
        System.out.println("Enter the elements of the array:");
        for (int i = 0; i < n; i++) {
            arr[i] = scanner.nextInt();
        }

        findDuplicates(arr);

        scanner.close();
    }
}
